nixos/tests/networking/router.nix

```
@@ -0,0 +1,82 @@
+{ networkd }: {config, pkgs, ... }:
+  let
+    inherit (pkgs) lib;
+    qemu-common = import ../../lib/qemu-common.nix { inherit (pkgs) lib pkgs; };
```

```suggestion
    qemu-common = import ../../lib/qemu-common.nix { inherit lib pkgs; };
```

seems a bit excessively clever to use pkgs.pkgs, but you can ignore this

---

true, I think I just missed that :see_no_evil: